oshmem/tools/wrappers/oshmem-c.pc.in

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2010      Cisco Systems, Inc.  All rights reserved.
+# Copyright (c) 2016      IBM Corporation.  All rights reserved.
+# Copyright (c) 2018      Research Organization for Information Science
+#                         and Technology (RIST).  All rights reserved.
+#
+Name: Open SHMEM
+Description: Open MPI based Open SHMEM implementation
+Version: @OSHMEM_VERSION@
+URL: http://www.open-mpi.org/
+#
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+includedir=@includedir@
+libdir=@libdir@
+pkgincludedir=@opalincludedir@
+#
+# Note that -lopen-pal and all the EXTRA_LIBS are only necessary when
+# static linking (they're pulled in by libopen-rte.so's implicit
+# dependencies), so only list these in Libs.private.
+#
+Libs: -L${libdir} @OMPI_PKG_CONFIG_LDFLAGS@ -loshmem -l@OMPI_LIBMPI_NAME@
```

Comment:
(minor) Does it make sense to create a `OMPI_LIBOSHMEM_NAME` to replace hardcoded `oshmem`?

Reply:
If this is the only issue left on this PR, I'd say that it's ready / done / can be merged.

@ggouaillardet @amaslenn ?

Reply:
Nothing apart from this one from my side.

Reply:
@ggouaillardet If you still want this one, I'd say you're clear to merge.